appveyor.yml

```diff
@@ -23,7 +23,7 @@ environment:
         TEST_PHP_SQL_SERVER: (local)\SQL2017
         SQL_INSTANCE: SQL2017
         PHP_VC: 15
-        PHP_MAJOR_VER: 7.2
+        PHP_MAJOR_VER: 7.3
         PHP_MINOR_VER: 11
         PHP_EXE_PATH: x64\Release_TS
         THREAD: ts
```

Comment:
PHP_MAJOR_VER below is still 7.1?

Reply:
Yes, for now, until after 7.4 is released. In that case we will change the matrix to drop the image VS 2015